tests/integration/012_environment/main.tf

```diff
@@ -24,6 +25,11 @@ resource "env0_environment" "example" {
   revision                   = "master"
 }
 
+data "env0_configuration_variable" "env_config_variable" {
```

Comment:
Return this environment configuration variable test, now that it's fully support in our TF provider